src/api/lookups/media/media.js

```diff
@@ -4,32 +4,24 @@ import configuration from '../../openAPIConfig';
 import {MediaFileServiceApiFactory} from 'webitel-sdk';
 import eventBus from "../../../utils/eventBus";
 import store from "../../../store/store";
+import {WebitelSDKItemDeleter, WebitelSDKListGetter} from "../../utils/apiControllers";
 
 const mediaService = new MediaFileServiceApiFactory
 (configuration, '', instance);
 
 const token = localStorage.getItem('access-token');
 const BASE_URL = process.env.VUE_APP_API_URL;
 
-export const getMediaList = async (page = 0, size = 10, search) => {
-    const domainId = store.state.userinfo.domainId || undefined;
-    if(search && search.slice(-1) !== '*') search += '*';
-    const defaultObject = {
-        name: '',
-        _isSelected: false,
-    };
+const defaultListObject = {
```

Comment:
unnecessary: name is automatically generated, _isSelected is default in class

Reply:
done